iOS_SDK/OneSignalSDK/UnitTests/Shadows/OneSignalClientOverrider.h

```diff
@@ -48,5 +48,7 @@
 +(void)disableExecuteRequestOverride:(BOOL)disable;
 +(NSArray<OneSignalRequest *> *)executedRequests;
 + (void)setMockResponseForRequest:(NSString *)request withResponse:(NSDictionary *)response;
++ (NSDictionary*)iosParamsResponse;
++ (void)setRemoteParamResponse:(NSDictionary *)params;
```

Comment:
`setRemoteParamResponse` should have `params` in plural
`setRemoteParamsResponse`?